README.md

```diff
@@ -189,26 +188,26 @@ Pylance provides users with the ability to customize their Python language suppo
         -   `true`
         -   `false` (default)
 
-```python
-@pytest.fixture()
-def my_fixture() -> str:
-    return "foo"
+    ```python
```

Comment:
Did you preview this change? I had to mess with a bunch to get github to render this correctly.

Reply:
> build the extension and see if it renders appropriately?

This should be unrelated to what the user sees in VS Code, right? I believe that text comes from https://github.com/microsoft/pyrx/blob/main/packages/vscode-pylance/README.md.

Reply:
No I think the preview renderer was faithful to what github did. Thanks for checking.